### DISCUSSION TITLE
JWT template variables

### DISCUSSION THREAD

**Discussion by CodingKonsti** [keygen-sh/community#44] [Q&A] [answered]
Is there a possibility to get a numeric date form a template variable when creating a new license with jwt?
If not is there a possibility to add a field to the default payload?

**Comment by ezekg** [comment, marked as answer]
We don't provide numeric date template variables, dates/times are only available in ISO8601 format. The only available template variables are the ones [listed here](https://keygen.sh/docs/api/cryptography/#cryptographic-keys-template-vars).

But you can add any field to the JWT payload during license creation by providing a custom `key` attribute, e.g.

```json
"{\"t\":1661973757}"
```

The provided `key` will be used in the JWT `claims`, instead of the default.
